sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/MetricStorageUtils.java

```diff
@@ -19,7 +20,12 @@ private MetricStorageUtils() {}
    * <p>Note: This mutates the result map.
    */
   static <T> void mergeInPlace(
-      Map<Attributes, T> result, Map<Attributes, T> toMerge, Aggregator<T> aggregator) {
+      @Nullable Map<Attributes, T> result,
```

Comment:
@jsuereth Is `Collections.emptyMap()` (so making `getLastAccumulation` not-nullable) an alternative? Or does null and empty mean something different?

Reply:
`result` MUST be a writable map.  I'm also curious where the `null` is coming from here, probably poor checking upstream.

Reply:
Looks like #3831 fixed this, sweet